jetty-server/src/main/config/modules/threadpool.mod

```diff
@@ -0,0 +1,20 @@
+[description]
+Enables the Server thread pool.
+
+[xml]
+etc/jetty-threadpool.xml
+
+[ini-template]
+
+### Server Thread Pool Configuration
+## Minimum Number of Threads
+#jetty.threadPool.minThreads=10
+
+## Maximum Number of Threads
+#jetty.threadPool.maxThreads=200
+
+## Thread Idle Timeout (in milliseconds)
+#jetty.threadPool.idleTimeout=60000
+
+## Detailed Dump Required?
```

Comment:
It's a bit weird the question mark at the end.
I would change it to "Whether to output a detailed dump" or something similar _not_ in a question form.